webauthn/createcredential-excludecredentials.https.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>WebAuthn navigator.credentials.create() excludeCredentials Tests</title>
+<link rel="author" title="Adam Powers" href="mailto:adam@fidoalliance.org">
+<link rel="help" href="https://w3c.github.io/webauthn/#iface-credential">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src=helpers.js></script>
+<body></body>
+<script>
+standardSetup(function() {
+    "use strict";
+
+    // bad excludeCredentials values
+    new CreateCredentialsTest({path: "options.publicKey. excludeCredentials", value: undefined}).runTest("Bad user:  excludeCredentials missing", new TypeError());
```

Comment:
Unexpected spaces in these `path`s

Reply:
Also: I think setting `excludeCredentials` to `undefined` is valid, actually. That's as if we didn't pass the array, which is legal. So I think line 15 goes away, and 16 + 17 have that space trimmed.

Reply:
fixed in 9a708e1